### ISSUE TITLE
[Bug] java.lang.NullPointerException in the WebApiLineMarkerProvider for firstSortOrder.compareTo

### ISSUE THREAD

**Issue by Taral-Patoliya** [magento/magento2-phpstorm-plugin#646]
**Describe the bug** (*)

```
java.lang.NullPointerException
	at com.magento.idea.magento2plugin.linemarker.php.WebApiLineMarkerProvider$WebApiRoutesCollector.lambda$sortRoutes$0(WebApiLineMarkerProvider.java:142)
	at java.base/java.util.TimSort.countRunAndMakeAscending(TimSort.java:355)
	at java.base/java.util.TimSort.sort(TimSort.java:220)
	at java.base/java.util.Arrays.sort(Arrays.java:1515)
	at java.base/java.util.ArrayList.sort(ArrayList.java:1750)
	at com.magento.idea.magento2plugin.linemarker.php.WebApiLineMarkerProvider$WebApiRoutesCollector.sortRoutes(WebApiLineMarkerProvider.java:137)
	at com.magento.idea.magento2plugin.linemarker.php.WebApiLineMarkerProvider$WebApiRoutesCollector.getRoutes(WebApiLineMarkerProvider.java:92)
	at com.magento.idea.magento2plugin.linemarker.php.WebApiLineMarkerProvider.collectSlowLineMarkers(WebApiLineMarkerProvider.java:50)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.queryProviders(LineMarkersPass.java:185)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.lambda$doCollectInformation$3(LineMarkersPass.java:90)
	at com.intellij.codeInsight.daemon.impl.Divider.divideInsideAndOutsideInOneRoot(Divider.java:81)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.doCollectInformation(LineMarkersPass.java:77)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:56)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$1(PassExecutorService.java:414)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1078)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:407)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:705)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:647)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:63)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:406)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:382)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:174)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:183)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:380)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:188)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)

```

**To Reproduce** (*)

**Expected behavior** (*)

**Screenshots**

**Please complete the following information:** (*)

 - OS: Mac OS X 11.5.2
 - PhpStorm/Intellij version: 2021.2.2
 - Plugin Version: 4.1.0

**Additional context**

**Comment by bohdan-harniuk** [comment]
Hello, @Taral-Patoliya!

Thank you for your report!
We'll try to deliver fix in the next bug-fix release!

Thank you!

**Comment by bohdan-harniuk** [comment]
Hello, @Taral-Patoliya!

The bug was successfully reproduced:

<img width="1272" alt="Screenshot 2021-11-30 at 17 33 51" src="https://user-images.githubusercontent.com/31848341/144088423-d814dbc9-59e2-4053-b668-98f4bf96fd76.png">

The fix will be available in the next bug-fix release: 4.2.1.

Thanks, Bohdan
